frappe/core/doctype/docfield/docfield.json

```diff
@@ -592,13 +593,20 @@
    "fieldname": "placeholder",
    "fieldtype": "Data",
    "label": "Placeholder"
+  },
+  {
+   "default": "0",
+   "depends_on": "eval:['Attach', 'Attach Image'].includes(doc.fieldtype)",
+   "fieldname": "make_attachment_public",
+   "fieldtype": "Check",
+   "label": "Make Attachment Public"
```

Comment:
```suggestion
   "label": "Make Attachment Public (by default)"
```

Reply:
Because the true meaning of the previous label only comes to bear if you _also_ forbid to toggle this button in custom js code.

Reply:
> Because the true meaning of the unmodified label only comes to bear if you _also_ forbid to toggle this button in custom js code.

I agree with you, thanks.